crates/pop-cli/tests/parachain.rs

```diff
@@ -3,58 +3,78 @@
 #![cfg(feature = "parachain")]
 
 use anyhow::Result;
-use assert_cmd::{cargo::cargo_bin, Command};
+use assert_cmd::cargo::cargo_bin;
 use pop_common::{find_free_port, templates::Template};
 use pop_parachains::Parachain;
-use std::{fs, path::Path, process::Command as Cmd};
+use std::{ffi::OsStr, fs, path::Path, process::Command, thread::sleep, time::Duration};
 use strum::VariantArray;
-use tokio::time::{sleep, Duration};
 
-/// Test the parachain lifecycle: new, build, up, call.
-#[tokio::test]
-async fn parachain_lifecycle() -> Result<()> {
-	let temp = tempfile::tempdir().unwrap();
```

Comment:
Results in cache being ineffective, specifically in relation to the build of the template. Separated out into its own test.

Reply:
I used `let temp = tempfile::tempdir()?`; because when running the test locally, the parachain folder isn’t cleaned up otherwise. But I see your point

Reply:
Can add an instruction at the end of the test to delete the folder, but then each time the test is run locally one needs to wait for a rebuild and then there is no caching.

Reply:
Hmmm, as it stands the relevant directory is not being cached, presumably as it is buried within the directory structure. https://github.com/r0gue-io/pop-cli/actions/runs/15146706684/job/42584088437

It would therefore need updating on the CI workflow to include so I have reverted the change in 59a0cd9c. Waiting ages for CI to complete can be a chore, but so can having local disk space eaten up unknowingly.